### DISCUSSION TITLE
Will this work without nvidia gpu?

### DISCUSSION THREAD

**Discussion by equinox1307** [JaKooLit/Fedora-Hyprland#133] [General]
Hi all,

just asking if this will also install the Nvidia drivers if no Nvidia GPU is detected. Tried finding out by going through the code but couldnt find an answer.  I got integrated Intel HD graphics. 

Thanks in advance!

**Comment by JaKooLit** [comment]
if you choose N on the installation question, wont execute the nvidia script

Obviously, it will install nvidia drivers if you choose that you have nvidia gpu
